client/src/styles/GlobalStyles.tsx

```diff
@@ -135,11 +135,38 @@ const globalStyles: SerializedStyles = css`
   }
 `
 
+const darkMode: SerializedStyles = css`
+  body {
+    filter: invert(100%) hue-rotate(180deg);
+  }
+
+  .prevent-hue-rotate {
+    filter: invert(100%) hue-rotate(180deg);
+  }
+`
+
 const GlobalStyles: React.FC = () => {
+  const [isDarkMode, setIsDarkMode] = React.useState(false)
```

Comment:
It would be cool to save this on localStorage and/or use the systems theme as well ([there is a media query for that](https://stackoverflow.com/a/52986538)).